<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>835</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>835</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>835</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(221, 221, 221);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QFrame" name="left_frame">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>41</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>500</width>
        <height>772</height>
       </size>
      </property>
      <property name="toolTip">
       <string extracomment="Predict disease from symptoms/signs"/>
      </property>
      <property name="toolTipDuration">
       <number>7</number>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 160, 156);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="lineEdit">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>41</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 300 11pt &quot;Noto Sans Regular&quot;;</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listWidget">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;Noto Sans&quot;;
color: rgb(0, 0, 0);</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>20</number>
           </property>
           <property name="rightMargin">
            <number>20</number>
           </property>
           <item>
            <widget class="QPushButton" name="clearBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 10pt &quot;Maven Pro&quot;;</string>
             </property>
             <property name="text">
              <string>Clear</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="predictBtn">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="toolTip">
              <string extracomment="Predict disease from symptoms/signs"/>
             </property>
             <property name="whatsThis">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="styleSheet">
              <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 10pt &quot;Maven Pro&quot;;</string>
             </property>
             <property name="text">
              <string>Predict</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="leftMargin">
            <number>84</number>
           </property>
           <property name="rightMargin">
            <number>50</number>
           </property>
           <item>
            <widget class="QProgressBar" name="progressBar">
             <property name="sizePolicy">
              <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>16</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>21</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">font: 500 11pt &quot;Maven Pro&quot;;</string>
             </property>
             <property name="value">
              <number>24</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="status_label">
           <property name="maximumSize">
            <size>
             <width>61</width>
             <height>18</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(29, 160, 156);
color: rgb(255, 255, 255);
font: 500 11pt &quot;Maven Pro&quot;;</string>
           </property>
           <property name="text">
            <string> Done!</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="middle_frame">
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 160, 156);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QTabWidget" name="results_tabWidget">
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 10pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="possibles_tab">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="final_tab">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <property name="spacing">
          <number>50</number>
         </property>
         <item>
          <widget class="QPushButton" name="exportBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 10pt &quot;Maven Pro&quot;;</string>
           </property>
           <property name="text">
            <string>Export</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 10pt &quot;Maven Pro&quot;;</string>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="stgBtn">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 10pt &quot;Maven Pro&quot;;</string>
           </property>
           <property name="text">
            <string>STG</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QFrame" name="right_frame">
      <property name="maximumSize">
       <size>
        <width>315</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(29, 160, 156);</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QTabWidget" name="supported_tabWidget">
         <property name="cursor">
          <cursorShape>ArrowCursor</cursorShape>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: rgb(255, 255, 255);
font: 11pt &quot;Segoe UI&quot;;</string>
         </property>
         <property name="currentIndex">
          <number>1</number>
         </property>
         <widget class="QWidget" name="diseases_tab">
          <attribute name="title">
           <string>Tab 1</string>
          </attribute>
         </widget>
         <widget class="QWidget" name="criteria_tab">
          <attribute name="title">
           <string>Tab 2</string>
          </attribute>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <property name="spacing">
          <number>2</number>
         </property>
         <property name="leftMargin">
          <number>5</number>
         </property>
         <item>
          <widget class="QLabel" name="logoLabel">
           <property name="minimumSize">
            <size>
             <width>51</width>
             <height>21</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>51</width>
             <height>21</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(29, 160, 156);
color: rgb(255, 255, 255);
font: 500 11pt &quot;Maven Pro&quot;;</string>
           </property>
           <property name="text">
            <string>  logo</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="email_label">
           <property name="maximumSize">
            <size>
             <width>141</width>
             <height>16</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: rgb(29, 160, 156);
color: rgb(255, 255, 255);
font: 500 11pt &quot;Maven Pro&quot;;</string>
           </property>
           <property name="text">
            <string>eden.tz@yahoo.com</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>35</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>16777215</width>
     <height>16777215</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>Maven Pro Medium</family>
     <pointsize>11</pointsize>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>ArrowCursor</cursorShape>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 500 11pt &quot;Maven Pro Medium&quot;;
color: rgb(255, 255, 255);
background-color: rgb(100, 100, 100);</string>
   </property>
   <widget class="QMenu" name="menu_MENU">
    <property name="font">
     <font>
      <family>Maven Pro Medium</family>
      <pointsize>11</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>&amp;Menu</string>
    </property>
    <addaction name="load_stg_action"/>
    <addaction name="settings_action"/>
    <addaction name="about_action"/>
   </widget>
   <addaction name="menu_MENU"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="toolTip">
    <string/>
   </property>
   <property name="toolTipDuration">
    <number>-1</number>
   </property>
  </widget>
  <action name="load_stg_action">
   <property name="text">
    <string>&amp;Update STG</string>
   </property>
   <property name="font">
    <font>
     <family>Maven Pro</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="settings_action">
   <property name="text">
    <string>&amp;Settings</string>
   </property>
   <property name="font">
    <font>
     <family>Maven Pro</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
  <action name="about_action">
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="font">
    <font>
     <family>Maven Pro</family>
     <pointsize>10</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
